HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/Report/Report.cshtml

```diff
@@ -0,0 +1,87 @@
+@using HelpMyStreet.Utils.Utils;
+@using HelpMyStreet.Utils.Extensions;
+
+@model HelpMyStreetFE.Models.Account.Report.ReportViewModel
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```

Comment:
To fit with the model used elsewhere, we should include this via npm install rather than linking out to an externally hosted version at runtime.